test/runtests.jl

```diff
@@ -4,7 +4,7 @@ using Compat
 
 const DATADIR = joinpath(@__DIR__, "data")
 
-pd = xp_parse(open(readstring, joinpath(DATADIR,"t_s1.txt")))
+pd = xp_parse(open(f -> read(f, String), joinpath(DATADIR,"t_s1.txt")))
```

Comment:
doesn't this need a higher compat min?

Reply:
Updated to 0.33 to accommodate Test module name changes